### ISSUE TITLE
Explore  MEMIT method to deal with CSAM requests.

### ISSUE THREAD

**Issue by ontocord** [LAION-AI/Open-Assistant#705]
We are training with certain safety data built in, but even then, this might not be successful to steer a model away from responding for  CSAM requests. One alternative to finetuning is MEMIT. https://github.com/kmeng01/memit. We have a 1.3B model that would be good for experimenting using this method to create canned responses when receving CSAM requests. We would want to have a few QA pairs we know work well. Then after the MEMIT edits for CSAM requests, we test on the same QA pairs to see if the memory eidt affected performance.

**Comment by mikichan007** [comment]
Hi! I'd like to work on this issue

**Comment by mikichan007** [comment]
Where can I find your 1.3B model and test set to experiment on?

**Comment by ontocord** [comment]
https://huggingface.co/Rallio67/chip_1.4B_instruct_alpha

**Comment by mikichan007** [comment]
Sorry, I accidentally unassigned myself. 

I did some experiments with this. How can I share the results?

**Comment by mikichan007** [comment]
Should I put my notebook in 'notebooks' directory?

**Comment by shahules786** [comment]
@mikichan007 Can you share some key insights from your experiments here for discussion?
